src/main/controllers/UserResultsController.ts

```diff
@@ -39,7 +39,7 @@ export class UserResultsController extends RootController {
         return super.post(req, res, 'user-details', {
           content: {
             user,
-            showDelete: this.canDeleteUser(req.session.user, user),
+            showManage: this.canManageUser(req.session.user, user),
```

Comment:
'showManage' sounds a bit weird. May be show rename it to 'canManage'?

Reply:
fixed